nntrainer/src/tensor.cpp

```diff
@@ -241,47 +242,61 @@ Tensor Tensor::add(Tensor const &m) const {
   return result;
 }
 
-Tensor Tensor::subtract(Tensor const &m) const {
+int Tensor::subtract_i(Tensor const &m) {
   if (dim.channel() != m.dim.channel() || dim.height() != m.dim.height() ||
```

Comment:
I suppose that it needs to implement this subtract_i function in LazyTensor as well.  Am I right? Cause I couldn't find any in this PR.

Reply:
I am planning to make an independent PR to register all `*_i` operations to make PR compact. Thanks for pointing out!